### DISCUSSION TITLE
isenabled on wms-catalog-item removed in v8?

### DISCUSSION THREAD

**Discussion by pvgenuchten** [TerriaJS/terriajs#6861] [Q&A] [answered]
why is 'isenabled' property removed from wms-catalog-item, i couldn't find any documentation on this

having a layer enabled at load time seems usefull

compare 
https://docs-v7.terria.io/guide/connecting-to-data/catalog-type-details/wms/ and
https://docs.terria.io/guide/connecting-to-data/catalog-type-details/wms/

got an error on load with this parameter

**Comment by nf-s** [comment, marked as answer]
Hi @pvgenuchten 

In version 8 we have a new `workbench` property in the init JSON file - which contains the IDs of catalog items that are in the workbench on load. For example

```json
{
  "homeCamera": {
    "north": -8,
    "east": 158,
    "south": -45,
    "west": 109
  },
  "catalog": [
    {
      "type": "csv",
      "name": "CSV Test",
      "id": "zrPtHVJcPi",
      "url": "test/incidents_notime.csv"
    }
  ],
  "workbench": [
    "zrPtHVJcPi"
  ]
}
```

Can be tested with the following link

http://ci.terria.io/main/#start={%22version%22:%228.0.0%22,%22initSources%22:[%7B%22homeCamera%22%3A%20%7B%22north%22%3A%20-8%2C%22east%22%3A%20158%2C%22south%22%3A%20-45%2C%22west%22%3A%20109%7D%2C%22catalog%22%3A%20%5B%7B%22type%22%3A%20%22csv%22%2C%22name%22%3A%20%22CSV%20Test%22%2C%22id%22%3A%20%22zrPtHVJcPi%22%2C%22url%22%3A%20%22test%2Fincidents_notime.csv%22%7D%5D%2C%22workbench%22%3A%20%5B%22zrPtHVJcPi%22%5D%7D]}

All properties in the version 8 init file can be found here https://docs.terria.io/guide/customizing/initialization-files/#init-file-properties.

Let me know if that fixes your problem!

I'm going to convert this issue to a GitHub discussion.

Thanks,
Nick

**Comment by nf-s** [comment]
I should also say that we have a Catalog Converter tool, to upgrade catalogs from version 7 to version 8 - https://catalog-converter.terria.io/

This will automatically add catalog items with `isEnabled: true` to the new `workbench` property

**Comment by pvgenuchten** [comment]
I fully missed the workbench option, thanx for pointing out…
